app/workers/schedulers/stale_issue_marker.rb

```diff
@@ -9,7 +9,7 @@ class StaleIssueMarker
     include SidekiqWorkerMixin
 
     def perform
-      enabled_repo_names.each { |r| ::StaleIssueMarker.perform_async(r) }
+      StaleIssueMarker.perform_async
```

Comment:
As a result of making use of the `search_issues` endpoint, this does end up going against the previous implementation in https://github.com/ManageIQ/miq_bot/pull/331 .

However, I think that the reduced API calls and less dense `perform` end up making that argument less valid when using this approach.  By filtering by label and date as part of the search filter, we can eliminate entire calls if said repo doesn't even have open issues.

Reply:
Seems to make this entire class not necessary anymore, but we can keep for now.